client/src/modules/dashboard/Events/pages/NewEventPage.tsx

```diff
@@ -60,6 +69,17 @@ export const NewEventPage: NextPageWithLayout<{
         title: `Event "${eventData.createEvent.name}" created!`,
         status: 'success',
       });
+
+      let hasChapterCalendar = chapterData?.dashboardChapter.has_calendar;
+      if (chapter_id !== chapterId) {
+        const { data: currentChapter } = await refetch({
+          chapterId: chapter_id,
+        });
+        hasChapterCalendar = currentChapter.dashboardChapter.has_calendar;
+      }
```

Comment:
The fact this is necessary, got me to thinking: why not use the same source of truth for chapter name when on `dashboard/chapters/[id]/new-event` and `dashboard/events/new`? Namely `admined_chapters`.

Reason being, we only care about three things: `id`, `name` and `has_calendar`. If `admined_chapters` gets `has_calendar` we don't need to make (let alone pass around) an additional query. As a side benefit, it should be possible to simplify `NewEventPage` a little.

What do you reckon?